INSTALL.md

```diff
@@ -95,13 +95,15 @@ test -f matrix_key.pem || ./bin/generate-keys -private-key matrix_key.pem
 
 Create config file, based on `dendrite-config.yaml`. Call it `dendrite.yaml`. Things that will need editing include *at least*:
 * `server_name`
-* `database/*`
+* `database/*` ( All lines in the database section must have the username and password of the user created with the `createuser` command above. eg:`dendrite:password@localhost` )
```

Comment:
```suggestion
* `database/*` (All lines in the database section must have the username and password of the user created with the `createuser` command above. eg:`dendrite:password@localhost`)
```

Last little thing :)

Reply:
I thought you said there must be a space after a `(` and did that

Reply:
Oh I misunderstood one of your above comments.

Reply:
Done!

Reply:
The one at the ending parentheses too please!